### PR TITLE
GH-2189: Learn More link takes you to Ghostery Image url instead of site

### PR DESCRIPTION
**Pull request by benstrumeyer**
* [x] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

- Replace `https://www.ghostery.com/products/plus` with `https://www.ghostery.com/pricing` in:
- `PromoModal.jsx`
- `Subscribe.jsx`
- Historical Stats CTA link in `Stats.jsx` (Also updated Historical Stats Learn more link in description)

85e305ed-9308-4156-a3ce-f0dc8914d64c
 
Ticket: https://ghostery.atlassian.net/browse/GH-2189

### CODE REVIEW

File: app/panel/components/Subscribe.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const Subscribe = ({ match }) => {
 			<div className="pitch-container">
 				<span className="pitch-text" dangerouslySetInnerHTML={{ __html: t('subscribe_pitch') }} />
 			</div>
-			<PanelToTabLink href="https://www.ghostery.com/products/plus?utm_source=gbe&utm_campaign=in_app_plus_lm">
+			<PanelToTabLink href="https://www.ghostery.com/pricing/">
```
Comment (wlycdgr) [inline app/panel/components/Subscribe.jsx:41]: Let's restore the utm params here

Comment (wlycdgr) [reply app/panel/components/Subscribe.jsx:41]: Also, let's update the url to use `GHOSTERY_BASE_URL` to link to stage/prod as appropriate (as per Tino)

File: _locales/en/messages.json [hunk 2]
```diff
@@ -2099,7 +2099,7 @@
 				"content": "</a>"
 			},
 			"link_start": {
-				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/products/plus?utm_source=gbe&utm_campaign=in_app_hstats_lm\">"
+				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/pricing?utm_source=gbe&utm_campaign=in_app_hstats_lm\">"
```
Comment (wlycdgr) [inline _locales/en/messages.json:2102]: Let's update this to use `GHOSTERY_BASE_URL` too, if you can figure out a way to do this that doesn't require too much reworking of the code. Maybe this link can be pulled out of the string and instead be implemented in the component as a `PanelToTabLink`?

File: app/panel/components/Subscribe.jsx (hunk 1 above)
Comment (benstrumeyer) [reply app/panel/components/Subscribe.jsx:41]: Restored and updated link!

File: _locales/en/messages.json (hunk 2 above)
Comment (benstrumeyer) [reply _locales/en/messages.json:2102]: Good idea. Refactored to use `PanelToTabLink`!

**Review by wlycdgr — APPROVED** [review]
LGTM

**Pull request merged**